jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -112,6 +120,8 @@ define([
                     if ($helpArea) {
                         // remove if it exists since layout-specific help text will be inserted
                         $helpArea.remove();
```

Comment:
Since we are using jQuery to remove the help template elements, are the changes to the event handler creation above necessary? It would be nice to keep them in the help template.

From https://api.jquery.com/remove/:
> Similar to .empty(), the .remove() method takes elements out of the DOM. Use .remove() when you want to remove the element itself, as well as everything inside it. In addition to the elements themselves, all bound events and jQuery data associated with the elements are removed. To remove the elements without removing data and events, use .detach() instead.